packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -67,7 +67,11 @@ export function write_tsconfig(config, cwd = process.cwd()) {
 				compilerOptions: {
 					// generated options
 					baseUrl: config_relative('.'),
-					paths: get_tsconfig_paths(config),
+					paths: {
+						// enables import autocomplete for generated types aliased path
+						'./$types': [`${config.outDir}/types/src/routes/$types`],
```

Comment:
Not sure if this is correct, I'm not sure what implications this has on typescript resolving these paths, since `./$types` resolved to a different d.ts file for each page. It will probably help with intellisense though (as you have shown). Need to think on this more when I have time and try it out.

Reply:
Appreciate you looking into this. I'm still new to typescript so I don't know the implications either.